main/main.pde

```diff
@@ -18,10 +18,6 @@ final float SHRINK_RATE = 1;//2;//5;
 final int MAX_PARTICLES = 100;
 final int SPAWN_DELAY = 50; //ms
 
-int[] x1 = {320, 335, 350, 360, 372, 378, 380, 378, 372, 360, 350, 335, 320, 305, 290, 280, 272, 262, 260, 262, 268, 280, 290, 305};
-int[] y1 = {180, 182, 190, 200, 215, 228, 240, 252, 270, 280, 290, 298, 300, 298, 290, 280, 270, 256, 240, 228, 215, 200, 190, 182};
-int backgroundCounter = 10;
-
```

Comment:
Why was this removed?

Reply:
We are now generating the particles at random points around the location where the user presses instead of having hardcoded values. You can see this on line 129
```
      system.addParticle(new PVector(random(x-50, x+50), random(y-50, y+50)));

```